DEVELOPER.md

```diff
@@ -130,6 +130,23 @@ The VLAN ID (vid) usually follow this numbering convention.
 42  MGMT
 ```
 
+##### uplink via tunnel
+
+if for some reason you are in need of an uplink via a "normal" internet connection, a wireguard
+tunnel can be an easy and safe solution. in that case add another vlan to the networks.yml
+
+```yml
+  - vid: 13
+    untagged: true            # option to don't tag this vlan - useful if the corerouter is plugged into a normal home router
+    name: uplink
+    role: ext
+    tunnel_wan_ip: 192.168.1.2/24   # put here the address and subnet of the corerouter inside the uplink network
+    tunnel_wan_gw: 192.168.1.1      # gateway of the uplink network
+    tunnel_connections: 2           # default value, number of different tunnels to create
+    tunnel_timeout: 600             # timeout in seconds after this the tunnel is destroyed and attempted to be rebuild
+    tunnel_mesh_prefix_ipv4: 10.31.142.120/32   # ip address for this side of the tunnel
```

Comment:
This config will not work. You have a "pool" of 1 address because of /32 but you want 2 tunnel_connections.